api_bitrise.py

```diff
@@ -0,0 +1,99 @@
+import sys
+import os
+
+import pandas as pd
+from datetime import timezone
+from sqlalchemy import func
+
+from lib.bitrise_conn import BitriseAPIClient
+from utils.datetime_utils import DatetimeUtils as dt
+from database import (
+    Database,
+    ReportBitriseBuildsCount
+)
+
+
+class Bitrise:
+
+    def __init__(self):
+        try:
+            BITRISE_HOST = os.environ['BITRISE_HOST']
+            self.client = BitriseAPIClient(BITRISE_HOST)
+            self.client.BITRISE_APP_SLUG = os.environ['BITRISE_APP_SLUG']
+            self.client.token = os.environ['BITRISE_TOKEN']
+        except KeyError:
+            print("ERROR: Missing bitrise env var")
+            sys.exit(1)
+
+    # API: Builds
+    def builds(self, past_date_timestamp):
+        return self.client.get_builds(self.client.BITRISE_APP_SLUG, past_date_timestamp) # noqa
+
+    def get_builds_range_date(self, after):
+        return self.client.get_builds_time(self.client.BITRISE_APP_SLUG, after)
+
+
+class BitriseClient(Bitrise):
+
+    def __init__(self):
+        super().__init__()
+        self.db = DatabaseBitrise()
+
+    def bitrise_builds_detailed_info(self):
+        # Read latest timestamp from database
+        after = self.get_latest_build_from_database()
```

Comment:
Same as above.  I also prefer to go from most important aspect to less important. Something like `database_latest_build()` or something similar.